hieradata_aws/class/staging/db_admin.yaml

```diff
@@ -1,6 +1,6 @@
 govuk_env_sync::tasks:
   "pull_ckan_production_daily":
-    ensure: "present"
+    ensure: "disabled"
```

Comment:
Should it be "absent"? Or is "disabled" equivalent? Looking at the history, it seems we've got examples of both uses.

Reply:
I should have grepped before asking:

https://github.com/alphagov/govuk-puppet/blob/101e060c2dc2ea62decd31f7911a8c296fbbd8a9/modules/govuk_env_sync/manifests/task.pp#L38-L40